### PR TITLE
fixed typos in example

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: source/xml/writing.md [hunk 1]
```diff
@@ -122,13 +122,14 @@ This could output:
 You can even mix these syntaxes:
 
     $ns = '{http://www.w3.org/2005/Atom}';
-    $writer->namespaceMap['http://www.w3.org/2005/Atom'] = '';
```
Comment (staabm) [inline source/xml/writing.md]: this fixes the `:` problem described in https://github.com/fruux/sabre-xml/pull/61#issuecomment-162186171

File: source/xml/writing.md [hunk 2]
```diff
@@ -122,13 +122,14 @@ This could output:
...
+    $writer->namespaceMap['http://www.w3.org/2005/Atom'] = null;
 
     $writer->write([
         $ns . 'feed' => [
             $ns . 'title' => 'Example Feed',
             [
                 'name' => $ns . 'link',
+                'value' => '',
```
Comment (staabm) [inline source/xml/writing.md]: this fixes

```
Fatal error: Uncaught exception 'InvalidArgumentException' with message 'When passing an array to ->write with numeric indices, every item must be an array containing the "name" and "value" key'
```

when running the example

Comment (evert) [inline source/xml/writing.md]: Hmm... I want to fix sabre/xml to allow for this case instead =)

Comment (staabm) [inline source/xml/writing.md]: Same might be true for the other case above ;)

Comment (evert) [inline source/xml/writing.md]: Alright, I'll fix that too. Good point!

**Comment by evert** [comment]
Won't fix because we'll fix sabre/xml to handle these cases instead.

**Pull request closed without merging**